gsk4/src/auto/gl_shader_node.rs

```diff
@@ -24,6 +24,26 @@ impl glib::StaticType for GLShaderNode {
 }
 
 impl GLShaderNode {
+    #[doc(alias = "gsk_gl_shader_node_new")]
+    pub fn new(
+        shader: &GLShader,
+        bounds: &graphene::Rect,
+        args: &glib::Bytes,
+        children: &[RenderNode],
+    ) -> GLShaderNode {
+        skip_assert_initialized!();
+        let n_children = children.len() as u32;
+        unsafe {
+            from_glib_full(ffi::gsk_gl_shader_node_new(
+                shader.to_glib_none().0,
+                bounds.to_glib_none().0,
+                args.to_glib_none().0,
+                children.to_glib_none().0,
```

Comment:
This and all the others are needlessly creating a temporary `Vec`, which we should fix at the same time here. `&[RenderNode]` is equivalent to `&[*mut gsk_sys::RenderNode]` i.e. `*const *mut gsk_sys::RenderNode`.

Reply:
That would need to be fixed on glib-rs side right ?

Reply:
Yes, that should be an easy fix there. The `ToGlibContainerFromSlice` trait impl